docs/source/endpoints/rules.md

```diff
@@ -0,0 +1,167 @@
+---
+html_meta:
+  "description": "Available content rules in a Plone site can be listed and queried by accessing the /@content-rules endpoint on any context. Access requires the ManageContentRules permission."
+  "property=og:description": "Available content rules in a Plone site can be listed and queried by accessing the /@content-rules endpoint on any context. Access requires the ManageContentRules permission."
+  "property=og:title": "Content rules"
+  "keywords": "Plone, plone.restapi, REST, API, ContentRules, Rules"
+---
+
+(rules)=
+
+# Content rules
+
+```{note}
+These docs are generated by code tests, therefore you will see some `test` rules appear here.
+```
+
+Available content rules in a Plone site can be listed and queried by accessing the `/@content-rules` endpoint on any context.
+Access requires the ManageContentRules permission.
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/rules_get.req
+```
+
+```{literalinclude} ../../src/plone/restapi/tests/http-examples/rules_get.resp
+:language: http
+```
+
+The API consumer can assign, unassign , enable, disable,
+(disable) apply to subfolders any of the rules available in the portal.
+
+| Verb    | URL              | Action                                             |
+| ------- | ---------------- | -------------------------------------------------- |
+| `POST`  | `/@content-rules/{rule-id}`| Add rule to context                                          |
+| `GET`   | `/@content-rules/`         | Get aquired, assignable and assigned rules                      |
+| `PATCH` | `/@content-rules/` with RAW Body | enable/disable/(disable) apply to subfolders/move down/up |
```

Comment:
See previous note about "(disable) apply".

Also what does "RAW Body" mean? Why is it uppercase?